cboard-org/cboard
# Fix/fix infinite loop on paste boards recursively

**_This PR_**
Avoid an infinite loop by pasting a board inside of itself or on a child board. 
Close #1616 

**Before this**
[Screen.Recording.2023-12-12.at.18.04.58.zip](https://github.com/cboard-org/cboard/files/13679768/Screen.Recording.2023-12-12.at.18.04.58.zip)

**_To Test_**

**A**
  1) Copy a board
  2) Open that board and paste the board inside.
  A new Board Should be created that is an exact copy of the board copied (Also the folders inside continue to be linked to the same folders of the copied board. This is an old behavior.)

**B**
 1) Copy a board that contains subfolders
 2) Open that board and paste the board inside of a subfolder.
  A new Board Should be created that is an exact copy of the board copied (Also the folders inside continue to be linked to the same folders of the copied board. This is an old behavior.)
